### ISSUE TITLE
[Reminder] Does not parse `.gradle` files. anymore

### ISSUE THREAD

**Issue by BoykoAlex** [openrewrite/rewrite-all#14]
This issues is only in the current snapshot and is the result of `DefaultProjectParser` refactoring - this is just a reminder not to forget this thing.

The switch to `OmniParser` from `ResourceParser` resulted in `.gradle` files not parsed because `OmniParser` does not have `GradleParser` therefore this parser with proper build script and settings classpaths needs to be added to the OmniParser. Addition of a parser is inconvenient because it adds a parser at the end which means `PlainTextParser` would intercept `.gradle` files...

**Comment by BoykoAlex** [comment]
I don't think that `OmniParser` should have `GradleParser` by default... I think it should be added from **Gradle Plugin** code with proper classpaths for build script and settings script parsing...

**Comment by shanman190** [comment]
Yeah, that's true. It's probably going to be pretty difficult to accurate get those classpath entries from anywhere other than the Gradle plugin. 

It should be also be noted though that after a release this morning, this issue is now present in the latest release.
